### PR TITLE
Client: Map initial right click options

### PR DESCRIPTION
**Pull request by psiinon**
## Overview
Adds the following right click options to the Client Map:

- Attack (disabled)
- Copy URLs to Clipboard
- Delete
- Open URL in Browser
- Show in Sites Tree

## Checklist
- [x] Update help
- [x] Update changelog
- [x] Run `./gradlew spotlessApply` for code formatting
- [ ] Write tests
- [ ] Check code coverage
- [x] Sign-off commits
- [x] Squash commits
- [x] Use a descriptive title

For more details, please refer to the [developer rules and guidelines](https://www.zaproxy.org/docs/developer/dev-rules-and-guidelines/).

### CODE REVIEW

File: addOns/client/src/main/java/org/zaproxy/addon/client/PopupMenuClientDelete.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    public PopupMenuClientDelete(ClientMapPanel clientMapPanel) {
+        super(Constant.messages.getString("client.tree.popup.delete"), clientMapPanel);
+    }
+
+    @Override
+    public void performAction(ActionEvent e) {
+        if (View.getSingleton()
+                        .showConfirmDialog(
+                                Constant.messages.getString("client.tree.popup.delete.confirm"))
+                == JFileChooser.APPROVE_OPTION) {
```
Comment (kingthorin) [inline addOns/client/src/main/java/org/zaproxy/addon/client/PopupMenuClientDelete.java:40]: Isn't there a more appropriate/generic constant?

File: addOns/client/src/main/javahelp/org/zaproxy/addon/client/resources/help/contents/client.html [hunk 2]
```diff
@@ -37,6 +37,28 @@ <h3>Client Map</h3>
...
+
+<h4>Open in Browser</h4>
+
+Opens the selected node in the chosen browser.
+This menu item is disabled if multiple nodes are selected.
+If the URL contains a fragment then this will be passed to the browser.
+
+<h4>Show in Sites Tree</h4>
+
+Opens the selected node in the Sites Tree.
```
Comment (kingthorin) [inline addOns/client/src/main/javahelp/org/zaproxy/addon/client/resources/help/contents/client.html:59]: Does this need a mention that the message m the sites tree is the latest and does not necessarily represent the message or action that added it to the client map?

File: addOns/client/src/main/java/org/zaproxy/addon/client/PopupMenuClientDelete.java (hunk 1 above)
Comment (psiinon) [reply addOns/client/src/main/java/org/zaproxy/addon/client/PopupMenuClientDelete.java:40]: Not that I'm aware of :/ Its the one we use everywhere else isnt it?

Comment (psiinon) [reply addOns/client/src/main/java/org/zaproxy/addon/client/PopupMenuClientDelete.java:40]: yes: `JOptionPane.OK_OPTION` 😁

**Comment by psiinon** [comment]
Done 😁

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by kingthorin — APPROVED** [review]

**Comment by kingthorin** [comment]
Thanks!

**Pull request merged**